scripts/version.js

```diff
@@ -14,7 +14,7 @@ title(`Bumping version to v${version}.`)
 
 const currentBranch = exec.pipe(`git branch --show-current`)
 
-if (currentBranch === 'master') {
+if (currentBranch === 'main' || currentBranch === 'master') {
```

Comment:
Why are we maintaining `main` and `master` here?

Reply:
This is to hopefully still work if someone has not changed the branch to main.

Reply:
That said, this file is currently deleted on master, so this change is no longer needed. Fixed in latest push.